python/python/glide/async_commands/redis_modules/json.py

```diff
@@ -0,0 +1,118 @@
+            'OK'  # Indicates successful setting of the value at path '$' in the key stored at 'doc'.
+        >>> await redisJson.get(client, "doc", "$")
+            "[{\"a\":1.0,\"b\":2}]"  # Returns the value at path '$' in the JSON document stored at 'doc'.
+        >>> json_get = await redisJson.get(client, "doc", "$")
+        >>> json.loads(json_get)
+            [{"a": 1.0, "b" :2}]
+        """
+from typing import List, Optional, Union, cast
+
+from glide.async_commands.core import ConditionalChange
+from glide.constants import TOK
+from glide.protobuf.redis_request_pb2 import RequestType
+from glide.redis_client import TRedisClient
+
+
+async def set(
+    client: TRedisClient,
+    key: str,
+    path: str,
+    value: str,
+    set_condition: Optional[ConditionalChange] = None,
+) -> Optional[TOK]:
+    """
+    Sets the JSON value at the specified `path` stored at `key`.
+
+    See https://redis.io/commands/json.set/ for more details.
+
+    Args:
+        client (TRedisClient): The Redis client to execute the command.
+        key (str): The key of the JSON document.
+        path (str): Represents the path within the JSON document where the value will be set.
+            The key will be modified only if `value` is added as the last child in the specified `path`, or if the specified `path` acts as the parent of a new child being added.
+        value (set): The value to set at the specific path, in JSON formatted str.
+        set_condition (Optional[ConditionalChange]): Set the value only if the given condition is met (within the key or path).
+            Equivalent to [`XX` | `NX`] in the Redis API. Defaults to None.
+
+    Returns:
+        Optional[TOK]: If the value is successfully set, returns OK.
+            If value isn't set because of `set_condition`, returns None.
+
+    Examples:
+        >>> from glide import json as redisJson
+        >>> import json
+        >>> value = {'a': 1.0, 'b': 2}
+        >>> json_str = json.dumps(value)
+        >>> await redisJson.set(client, "doc", "$", json_str)
+            'OK'  # Indicates successful setting of the value at path '$' in the key stored at `doc`.
+    """
+    args = [key, path, value]
+    if set_condition:
+        args.append(set_condition.value)
+
+    return cast(Optional[TOK], await client._execute_command(RequestType.JsonSet, args))
+
+
+async def get(
+    client: TRedisClient,
+    key: str,
+    paths: Optional[Union[str, List[str]]] = None,
+    indent: Optional[str] = None,
+    newline: Optional[str] = None,
+    space: Optional[str] = None,
+) -> Optional[str]:
+    """
+    Retrieves the JSON value at the specified `paths` stored at `key`.
+
+    See https://redis.io/commands/json.get/ for more details.
+
+    Args:
+        client (TRedisClient): The Redis client to execute the command.
+        key (str): The key of the JSON document.
+        paths (Optional[Union[str, List[str]]]): The path or list of paths within the JSON document.
```

Comment:
lets write that the default is root, it feels not readable enough without this info